types/pg-pool/index.d.ts

```diff
@@ -12,7 +12,7 @@ declare class Pool<T extends pg.Client> extends pg.Pool {
     constructor(config?: Pool.Config<T>, client?: Pool.ClientLikeCtr<T>);
 
     connect(): Promise<T & pg.PoolClient>;
-    connect(callback: (err: Error, client: T & pg.PoolClient, done: (release?: any) => void) => void): void;
+    connect(callback: (err?: Error, client?: T & pg.PoolClient, done?: (release?: any) => void) => void): void;
```

Comment:
Based on your references I don't see a reason why `done` should be optional.

Reply:
I just noticed that it's marked as optional in the [pg documentation](https://node-postgres.com/api/pool#pool.connect), but it's never explained when this is the case and why.
As far as I can see there aren't any null-checks in the tests you referenced either.

Reply:
https://github.com/brianc/node-postgres/blob/master/packages/pg-pool/index.js#L161
The line number changed since the last PR. Please reference the above for an undefined done

Reply:
Ah yes, that makes a lot more sense.
So if I understand correctly, the `done` callback would be `undefined` if `err` is not? Given that we **have** to call it according to the documentation, I think it wouldn't make sense to "not have it", unless there's an error (like at your reference).

Also, based on [Pool's documentation](https://node-postgres.com/api/pool#releaseCallback), shouldn't the `done` parameter be called `release` and the `release` parameter inside `done` be called `error`?